libraries/Microsoft.Bot.StreamingExtensions/HttpContentStream.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// An attachment contained within a <see cref="Request"/>'s stream collection,
+    /// which itself contains any form of media item.
+    /// </summary>
+    public class HttpContentStream
+    {
```

Comment:
This class is not a stream. The name is very misleading. Please revise.

Reply:
No, it is the container for how you get a reference to the actual stream, along with the length and mime content type on ReceiveRequests and ReceiveResponses.